cmake/BackwardsCompatibility.cmake

```diff
@@ -0,0 +1,16 @@
+# REMOVE THIS FILE AS SOON AS TESTS ARE FIXED
```

Comment:
Nice workaround to separate the expected from any unexpected regressions!

Reply:
We just have to remember deleting it :joy: